### PR TITLE
Add tolerance when comparing images

### PR DESCRIPTION
**Pull request by adela-istrate-hs**

### CODE REVIEW

**Comment by adela-istrate-hs** [comment]
@valentin-radulescu-hs @flavius-tirnacop-hs review?

File: lib/mugshot.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ var LooksSameAdapter = require('./adapters/looks-same.js');
  * @type {Differ}
  * @const
  */
-var DEFAULT_DIFFER = new LooksSameAdapter();
+var DEFAULT_DIFFER = new LooksSameAdapter({tolerance: 1});
```
Comment (flavius-tirnacop-hs) [inline lib/mugshot.js]: Can you add a comment to explain what does 1 represent? Is it 1%?

**Comment by flavius-tirnacop-hs** [comment]
LGTM

File: lib/mugshot.js (hunk 1 above)
Comment (adela-istrate-hs) [inline lib/mugshot.js]: It is 1%, but we decided to drop this and let the user choose what tolerance is needed.

**Comment by valentin-radulescu-hs** [comment]
@adela-istrate-hs can we close this PR without merging?

**Pull request closed without merging**